javascript/timer.js

```diff
@@ -0,0 +1,43 @@
+var timer = document.getElementById("timer");
```

Comment:
use let instead of var

refer to these guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0